apps/nextjs-website/react-components/types/Form/Form.types.ts

```diff
@@ -3,17 +3,38 @@ import { CtaButtonProps, type CommonProps } from '../common/Common.types';
 export interface FormProps extends CommonProps {
   readonly title: string;
   readonly subtitle: string;
-  readonly privacyText: string | JSX.Element;
-  readonly privacyLink: string;
-  readonly privacyLinkText: string;
+  readonly privacyLinkRecaptchaPolicy: string;
+  readonly privacyLinkTextRecaptchaPolicy: string;
+  readonly privacyLinkRecaptchaTerms: string;
+  readonly privacyLinkTextRecaptchaTerms: string;
   readonly theme: 'light' | 'dark';
   readonly backgroundImage?: string;
   readonly ctaButtons: ReadonlyArray<CtaButtonProps>;
+  readonly showFirstName?: boolean;
+  readonly showLastName?: boolean;
+  readonly showEmail?: boolean;
+  readonly showOrganization?: boolean;
+  readonly showCitizen?: boolean;
+  readonly showPublicEmployee?: boolean;
+  readonly showTechPartner?: boolean;
+  readonly showDeveloper?: boolean;
+  readonly showJournalist?: boolean;
+  readonly checkboxTitle?: string;
+  readonly showCheckboxInfo?: boolean;
+  readonly showPrivacyDisclaimer?: boolean;
+  readonly privacyDisclaimerText?: string;
 }
 
+
+
 export interface FormData {
-  firstName: string;
-  lastName: string;
+  firstName?: string;
+  lastName?: string;
   email: string;
-  organization: string;
-};
\ No newline at end of file
+  organization?: string;
+  citizen: boolean;
+  publicEmployee: boolean;
+  techPartner: boolean;
+  developer: boolean;
+  journalist: boolean;
```

Comment:
Is it really ok for us to return `false` on values the user did not input?

I personally would have kept these optional to be able to distinguish between:
- undefined: User was not presented a choice
- false: User chose NOT to tick the box
- true: User chose to tick the box